posthog/models/notebook/notebook.py

```diff
@@ -0,0 +1,27 @@
+from django.db.models import JSONField
+from django.utils import timezone
+
+from django.db import models
+
+from posthog.models.utils import UUIDModel
+from posthog.utils import generate_short_id
+
+
+class Notebook(UUIDModel):
+    short_id: models.CharField = models.CharField(max_length=12, blank=True, default=generate_short_id)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+
+    title: models.CharField = models.CharField(max_length=400)
```

Comment:
Maybe, maybe not relevant - in theory, notebooks don't need to have an explicit title... The title is just the first line of the notebook...

We would however want to make the "content" searchable which may involve creating a linked field like "content_string" to actually search throughout the entire notebook... I dunno. Needs some thought but given how complex modifying DB fields are, could we perhaps start _without_ `title` and then add our searchable value once we have a better feel for what that looks like?

Reply:
> The title is just the first line of the notebook...

🤯 doh